exercises/react/canonical-data.json

```diff
@@ -46,313 +46,340 @@
     {
       "description": "input cells have a value",
       "property": "react",
-      "cells": [
-        {
-          "name": "input",
-          "type": "input",
-          "initial_value": 10
-        }
-      ],
-      "operations": [
-        {
-          "type": "expect_cell_value",
-          "cell": "input",
-          "value": 10
-        }
-      ]
+      "input": {
+        "cells": [
+          {
+            "name": "input",
+            "type": "input",
+            "initial_value": 10
+          }
+        ],
+        "operations": [
+          {
+            "type": "expect_cell_value",
```

Comment:
In its current state, it is not possible to **consistently** place `expect_cell_value` in `expected`, so I do not.